rest/config.go

```diff
@@ -283,9 +283,10 @@ type ChannelCacheConfig struct {
 }
 
 type UnsupportedServerConfig struct {
-	Http2Config           *Http2Config `json:"http2,omitempty"`               // Config settings for HTTP2
-	StatsLogFrequencySecs *uint        `json:"stats_log_freq_secs,omitempty"` // How often should stats be written to stats logs
-	UseStdlibJSON         *bool        `json:"use_stdlib_json,omitempty"`     // Bypass the jsoniter package and use Go's stdlib instead
+	Http2Config           *Http2Config           `json:"http2,omitempty"`               // Config settings for HTTP2
+	StatsLogFrequencySecs *uint                  `json:"stats_log_freq_secs,omitempty"` // How often should stats be written to stats logs
+	UseStdlibJSON         *bool                  `json:"use_stdlib_json,omitempty"`     // Bypass the jsoniter package and use Go's stdlib instead
+	SSLClientCerts        *base.ClientCertConfig `json:",omitempty"`                    // SSL/TLS _client_ certificate config
```

Comment:
Should probably be changed to `json:"ssl_client_certs,omitempty` for consistency.

Reply:
I made it consistent with the `SSLCert` and `SSLKey` keys, but if you want new keys to use snake_case I can change it.